Gopkg.toml

```diff
@@ -63,7 +63,7 @@ required = [
 
 [[override]]
   name = "github.com/cloudevents/sdk-go"
-  version = "0.10.0"
+  branch = "master"
```

Comment:
Can you add a comment explaining why this is master and when it can be moved back to a released version?

Reply:
sure, my bad. Let me add that in a follow up... Thanks!

Reply:
Done! https://github.com/google/knative-gcp/pull/449